packaging/fedora/build

```diff
@@ -19,25 +19,29 @@ import time
 
 CH_BASE = os.path.abspath(os.path.dirname(__file__) + "/../..")
 CH_RUN  = [CH_BASE + "/bin/ch-run"]
-PACKAGES = ["charliecloud", "charliecloud-debuginfo", "charliecloud-test"]
+PACKAGES = ["charliecloud",
+            "charliecloud-debuginfo",
+            "charliecloud-doc",
+            "charliecloud-test"]
 ARCH = platform.machine()
 
 def main():
 
    # Parse arguments.
    ap = argparse.ArgumentParser()
-   ap.add_argument("version")
-   ap.add_argument("--image", metavar="DIR")
+   ap.add_argument("image", metavar="IMAGE")
+   ap.add_argument("version"),
    ap.add_argument("--install", action="store_true")
    ap.add_argument("--rpmbuild", metavar="DIR",
                    default="%s/rpmbuild" % os.getenv("HOME"))
    args = ap.parse_args()
    print("# Charliecloud root:  %s" % CH_BASE)
-   print("""\
-# version:            %(version)s
-# image:              %(image)s
-# install:            %(install)s
-# rpmbuild root:      %(rpmbuild)s""" % args.__dict__)
+  # print("""\
+# version:            %(args.version)s
+# image:              %(args.image)s
+# install:            %(args.install)s
+# rpmbuild root:      %(args.rpmbuild)s
+# architecture:       %s(ARCH)""" % locals())
```

Comment:
Should this still be commented out?

Reply:
It doesn't work with `locals()`. I can revert it back to two less tidy print statements.

Reply:
Yes, better to have uglier print calls than no status output IMO.